src/commons/sagas/RequestsSaga.ts

```diff
@@ -224,15 +218,11 @@ export async function bulkUpdateAchievements(
   achievements: AchievementItem[],
   tokens: Tokens
 ): Promise<Response | null> {
-  const resp = await request(`admin/achievements`, 'PUT', {
-    accessToken: tokens.accessToken,
-    body: { achievements: achievements },
-    noHeaderAccept: true,
-    refreshToken: tokens.refreshToken,
-    shouldAutoLogout: false,
-    shouldRefresh: true
-  });
-
+  // TODO
+  const resp = await Cadet.adminAchievements.bulkUpdate(
+    (achievements as unknown) as Achievement[],
```

Comment:
There's still quite a bit of typecasting around the file, as there are still slight differences between the auto-generated types and the handwritten types. I'll look into all of these cases later.